module-audio/Audio/decoder/Decoder.hpp

```diff
@@ -16,19 +16,19 @@ namespace audio
 {
     namespace channel
     {
-        constexpr inline auto monoSound   = 1;
-        constexpr inline auto stereoSound = 2;
+        inline constexpr auto monoSound   = 1;
+        inline constexpr auto stereoSound = 2;
```

Comment:
Let's consider using `enum` here ;)

Reply:
I thought about it, but these values are used as an absolute numbers in `pushAudioData`:

```
const auto readScale     = (channelMode == ChannelMode::ForceStereo) ? channel::stereoSound : channel::monoSound;
```
 and I would have to `static_cast` them anyway - IMO this will look more complicated.

Reply:
Thanks for the explanation ;) It makes sense